Source/WebKit/UIProcess/Cocoa/UserMediaCaptureManagerProxy.cpp

```diff
@@ -521,6 +526,12 @@ void UserMediaCaptureManagerProxy::clone(RealtimeMediaSourceIdentifier clonedID,
     }
 }
 
+void UserMediaCaptureManagerProxy::getPhotoCapabilities(RealtimeMediaSourceIdentifier sourceID, GetPhotoCapabilitiesCallback&& handler)
+{
+    if (auto* proxy = m_proxies.get(sourceID))
+        proxy->getPhotoCapabilities(WTFMove(handler));
```

Comment:
need to call the handler with error here in case there's no proxy

Reply:
Oh, good point!